docs/reference/commandline/images.md

```diff
@@ -52,6 +52,35 @@ uses up the `VIRTUAL SIZE` listed only once.
     postgres                  9.3.5               746b819f315e        4 days ago          213.4 MB
     postgres                  latest              746b819f315e        4 days ago          213.4 MB
 
+### Listing images by name and tag
+
+The ``docker images`` command takes an optional ``[REPOSITORY[:TAG]]``
```

Comment:
Just a note: wondering why you're using multiple back-ticks; in general we're using single back-ticks for this (but this should probably work as well) :D